templates/geth/geth.py

```diff
@@ -0,0 +1,243 @@
+import os
+import time
+import io
+import requests
+from random import shuffle
+
+from jumpscale import j
+from zerorobot.service_collection import ServiceNotFoundError
+from zerorobot.template.base import TemplateBase
+from zerorobot.template.decorator import retry
+from zerorobot.template.state import StateCheckError
+
+ALT_STARTUP_TEMPLATE = """
+
+[startup.geth]
+name = "core.system"
+
+[startup.geth.args]
+name = "/sandbox/bin/geth"
+args = ["--{GETH_NETWORK}", "--verbosity={GETH_VERBOSITY}",
+    "--lightserv={GETH_LIGHTSERV}", "--nat={GET_NAT}", "--{GETH_V5DISC}", 
+    "--syncmode={GETH_SYNCMODE}", "--datadir={GETH_DATADIR}", "--rpc", 
+    "--rpcaddr=0.0.0.0", "--ethport={GETH_PORT}"]
+"""
+
+class Geth(TemplateBase):
+    version = '0.0.9'
```

Comment:
please change this version back to `0.0.2` so it is in line with the other templates in this repo. It is possible to update a service without changing the version